src/sentry/static/sentry/app/components/group/externalIssueActions.jsx

```diff
@@ -165,16 +176,24 @@ class ExternalIssueForm extends AsyncComponent {
         submitLabel={SUBMIT_LABEL_BY_ACTION[action]}
         footerClass="modal-footer"
       >
-        {config.map(field => (
-          <FieldFromConfig
-            key={field.name}
-            field={field}
-            inline={false}
-            stacked
-            flexibleControlStateSize
-            {...this.getFieldProps(field)}
-          />
-        ))}
+        <Accordion
+          cutoff={3}
+          open={this.state.showAllFields}
+          style={{margin: '1em 0'}}
```

Comment:
Where does this style get applied? I don't know if this is actually used?

Reply:
yeah good call. looks like this lost it's purpose when i refactored.